docs/source/roadmap.md

```diff
@@ -0,0 +1,287 @@
+3. Cell x Gene table can be easily imported by computational biologists (outside of _starfish_) to determine cell type or cluster membership for each cell.  This cluster membership can include probabilistic membership to multiple clusters and is added to the Cell x Gene table as metadata
+
+#### A tool to convert image data into SpaceTx Format
+1. CLI and API (Java) can parse [supported file formats](https://docs.openmicroscopy.org/bio-formats/6.0.0-m3/supported-formats.html) into SpaceTx-formatted datasets<sup>[5](#fn5)</sup>
+2. Installation of the tool can be accomplished by downloading an archive with a launcher script or by using pre-built docker images
+3. Users and/or data wranglers can point the tool at one of the [stated files](https://docs.openmicroscopy.org/bio-formats/6.0.0-m3/formats/dataset-table.html) for each FOV to generate 2D TIFFs as well as the necessary SpaceTX JSON files
+4. In the case that multiple independent images (a “series”) are present in a single dataset, the user must specify which offset to use
+5. Multiple datasets can be passed in which case each in order is considered a separate field of view
+5. The starting index of the field of view may be changed from the default of zero
+6. If a codebook exists at the time of creation, it can be passed to the tool for inclusion in the JSON. Otherwise, a dummy codebook will be created
+
+
+## Candidate features for future releases
+
+There are many other features that are good candidates for future releases. We discuss these features, and why we believe they can be built at a later point.
+
+### Hardware- and Acquisition-related corrections
+
+We lack general purpose solutions for registration that fulfill requirements for ISS (the Harris version), smFISH, & MERFISH. Additionally, expansion microscopy requires non-affine thin-plate spline registration. Registration is out of scope for the timeline expected by SpaceTx to have all the data processed.
+
+In order to benchmark datasets, all contributors will need to provide pre-processed images before uploading. This will help us focus on _starfish_'s key value proposition and meet our goal of comparing the results from all the methods using standard file formats, and scalable, reproducible image processing workflows.
+
+A key challenge in developing generalized solutions to registration and other image processing corrections is that the necessary corrections are highly specific to the signal acquisition challenges each data generator faces. Furthermore, proper image pre-processing is critical to the success of the downstream pipeline components that we're building in the roadmap defined above. Because data generators understand their hardware, chemistry, and samples much better than we do, for now, they are best able to apply their solutions before the data goes into _starfish_ pipelines.
```

Comment:
@ambrosejcarr I think we can have the field flatness/image normalization (`ElementWiseMult()`) and the `LinearUnmixing` components landed by the 0.1.0 date (April 2019).